spirv_cross.hpp

```diff
@@ -612,6 +612,9 @@ class Compiler
 	void register_read(uint32_t expr, uint32_t chain, bool forwarded);
 	void register_write(uint32_t chain);
 
+	// Returns true if the target language supports combined texture-samplers. Returns fasle by default.
+	virtual bool supports_combined_samplers() const;
```

Comment:
This needs to be moved to spirv_glsl.hpp, virtual backend methods do it belong here.

Reply:
This function is used by `Compiler::CombinedImageSamplerUsageHandler::add_hierarchy_to_comparison_ids` and a few others in `spirv_cross.cpp`. What do you recommend to move it outside of the `Compiler` class?

Reply:
In that case, it's best to just pass a bool into CombinedImageSamplerUsageHandler if it's supported or not. The base class should not know about backend specifics.